includes/cs-main-frontend.php

```diff
@@ -48,6 +50,16 @@
 		<?php esc_html_e( 'Global Settings', 'advanced-currency-switcher' ); ?>
 	</a>
 
+	<a href="?page=currency_switch&tab=style" class="nav-tab tb
+	<?php
+	if ( 'style' === $active_tab ) {
+		echo 'nav-tab-active';
+	}
+	?>
+	">
+		<?php esc_html_e( 'Style', 'advanced-currency-switcher' ); ?>
+	</a>
+
 	<a href="?page=currency_switch&tab=user-manual" class="nav-tab tb
```

Comment:
Can we use https://developer.wordpress.org/reference/functions/add_query_arg/ here?